deploy.sh

```
@@ -0,0 +1,68 @@
+
+if [ $? -ne 0 ] || [ -z "$SECRET_JSON" ]; then
+    echo "ERROR: Secrets Manager 정보 로드 실패. IAM 권한/Secret ID를 확인하세요." >&2
+    exit 1
+fi
+
+# --- 환경 변수 설정 (쉘에 Export) ---
+echo "--- 2/4: 환경 변수 설정 ---"
+
+# 쉘 환경 변수로 export (jq로 JSON 값 추출)
+# 모든 ${변수명}을 Secrets Manager의 JSON 키와 매칭하여 정의해야 함
+export RDS_URL=$(echo "$SECRET_JSON" | jq -r '.RDS_URL')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)